doc/source/cluster/cluster_under_construction/references/index.md

```diff
@@ -9,6 +9,5 @@ Refer to the {ref}`KubeRay documentation<kuberay-api-reference>` or {ref}`Ray Cl
 :name: ray-clusters-reference
 
 autoscaler-sdk-api
```

Comment:
Do you think we should also move this page? [This](https://docs.ray.io/en/master/cluster/cluster_under_construction/references/autoscaler-sdk-api.html) is what it looks like in the compiled docs. If so we can delete the `cluster/cluster_under_construction/references` section completely.

Reply:
Yep, good idea. I'll do this in a follow-up PR for addressing the autoscaler changes.